src/page.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import Counter from './counter';
+import Buttons from './buttons';
```

Comment:
```suggestion
import React, { useState } from 'react';

import Counter from './counter';
import Buttons from './buttons';
```

의미 상으로 분리되는 경우 공백을 넣어서 구분해주시면 좋아요.
빈 코드 한줄에도 다 의미를 담아서 작성하신다고 생각하시면 좋아요.